docs/devel/develop-with-minikube.rst

```diff
@@ -30,6 +30,27 @@ First, run minikube, and configure your local kubectl command to work with minik
    $ helm init
 
 
+Install local development tools
+===============================
+
+You will need the following tools to build cert-manager:
+
+* Go_ (You may need a newer version than ships with your distro)
+* Bazel_
+* Docker_ (and enable for non-root user)
+
+These instructions have only been tested on Linux; Windows and MacOS may
+require further changes.
+
+If you need to add dependencies, you will additionally need:
+
+* Dep_
+* Gazelle_
+* Kazel_
+* Buildozer_
```

Comment:
Bazel should take care of grabbing `dep`, `gazelle`, `kazel` and `buildozer` if you run the `update-deps` target with `bazel run //hack:update-deps`

Reply:
I found I needed git (assumed) and mercurial in order for `dep` to succeed, even with `bazel`. Otherwise, I got:

```
^CSignal received: waiting for 1 ops to complete...
grouped write of manifest, lock and vendor: failed to export bitbucket.org/ww/goautoneg: 
        (1) hg is not installed: 
        (2) hg is not installed: 
        (3) hg is not installed: 
        (4) failed to list versions for https://bitbucket.org/ww/goautoneg: remote: Mercurial (hg) is required to use this repository.
remote: 
remote: https://confluence.atlassian.com/x/vLRCMw
fatal: unable to access 'https://bitbucket.org/ww/goautoneg/': The requested URL returned error: 405
: exit status 128
        (5) failed to list versions for ssh://git@bitbucket.org/ww/goautoneg: : signal: killed
        (6) context canceled
        (7) context canceled
```